IMAGE:
  WIDTH: 640
  HEIGHT: 480
  ROI_START_HEIGHT: 385
  ROI_HEIGHT: 40

XYCAR:
  START_SPEED: 0.0
  MAX_SPEED: 4.0
  MIN_SPEED: 4.0
  SPEED_CONTROL_THRESHOLD: 15.0
  ACCELERATION_STEP: 0.05
  DECELERATION_STEP: 0.1
  INIT_IS_LANEKEEPING: false
  INIT_IS_STOPLINE: true
  INIT_IS_REDLIGHT: true

PID:
  P_GAIN: 0.53
  I_GAIN: 0.00251
  D_GAIN: 0.3

MOVING_AVERAGE_FILTER:
  SAMPLE_SIZE: 30

LIDAR:
  DISTANCE_M: 0.32
  CHECK_THRESHOLD: 3
  CHECK_ANGLE: 10
  AVOID_ANGLE: 30
  AVOID_DURATION1: 3.0
  AVOID_DURATION2: 4.0
  AVOID_LOOPCNT_ANGLE: 2
  AVOID_LOOPCNT_SIDE: 2
  AVOID_LOOPCNT_FOWORD: 4
  AVOID_SPEED: 3.0

STOP:
  NON_ZERO_COUNT: 650
  SLEEP_TIME: 700000

TSTL:
  BEV_BLOCK_SIZE: 90
  DISTANCE_LOW_THRESHOLD : 50
  DISTANCE_HIGH_THRESHOLD : 130
  DRIVE_ANGLE : 50

TOPIC:
  PUB_NAME: /xycar_motor
  SUB_NAME: /usb_cam/image_raw/
  DETECT_SUB_NAME : /yolov3_trt_ros/trt_detection
  LIDAR_SUB_NAME: /scan
  QUEUE_SIZE: 1

CANNY:
  LOW_THRESHOLD: 150
  HIGH_THRESHOLD: 250

HOUGH:
  ABS_SLOPE_RANGE: 10.0
  THRESHOLD: 30
  MIN_LINE_LENGTH: 30
  MAX_LINE_GAP: 10

DEBUG: false